### PR TITLE
Adds lightline support

### PR DESCRIPTION
**Pull request by megalithic**
Please give this a look and see if anything looks out of whack to you. Thanks for your awesome color scheme/project @trevordmiller!

screenshot

### CODE REVIEW

File: src/lightline.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
+const colorValues = require("nova-colors").default;
+const uiGroups = require("nova-colors").uiGroups;
+
+/* COLOR REFERENCES:
```
Comment (trevordmiller) [inline src/lightline.js]: Please remove this comment block so it doesn't get out of sync with the `nova-colors` code :)

Comment (trevordmiller) [reply src/lightline.js]: Please also include screenshot(s) if you don't mind :)

**Comment by megalithic** [comment]
@trevordmiller done! thanks much!

**Comment by megalithic** [comment]
@trevordmiller .. done!

**Comment by trevordmiller** [comment]
@megalithic Awesome, thanks! I should be able to get this released sometime in the next week.

**Comment by trevordmiller** [comment]
This has been released in `3.21.0`: https://github.com/trevordmiller/nova-vim/releases

**Comment by megalithic** [comment]
thanks much @trevordmiller

**Comment by trevordmiller** [comment]
@megalithic np - please upgrade and let me know if you have any problems :) Thanks for using Nova!

**Pull request merged**